include/deal.II/distributed/tria_base.h

```diff
@@ -179,28 +181,28 @@ namespace parallel
        * This vector stores the number of locally owned active cells per MPI
        * rank.
        */
-      std::vector<unsigned int> n_locally_owned_active_cells;
```

Comment:
I'm just aligning the names of member variables above and below to make it easier on the eye. I don't think we have a convention, but we do it in many places.